gdk-pixbuf/Gir.toml

```
@@ -52,20 +52,20 @@ status = "generate"
     ignore = true
     [[object.function]]
     name = "new_from_file"
-    #manual is_windows_utf8
-    manual = true
+        [object.function.return]
+        nullable = false
     [[object.function]]
     name = "new"
         [object.function.return]
-        nullable = true
+        nullable = false
```

```suggestion
        nullable = true
```

This single one should stay `true`. The others are fine